applications/ChimeraApplication/custom_utilities/fractional_step_settings_for_chimera.h

```diff
@@ -0,0 +1,289 @@
+
+    ///@}
+    ///@name Access
+    ///@{
+
+    void SetStrategy(StrategyLabel const& rStrategyLabel,
+                             typename TLinearSolver::Pointer pLinearSolver,
+                             const double Tolerance,
+                             const unsigned int MaxIter) override
+    {
+        KRATOS_TRY;
+
+        // pointer types for solution strategy construction
+        typedef typename Scheme< TSparseSpace, TDenseSpace >::Pointer SchemePointerType;
+        typedef ResidualBasedBlockBuilderAndSolverWithConstraintsForChimera<TSparseSpace, TDenseSpace, TLinearSolver > ResidualBasedBlockBuilderAndSolverWithConstraintsForChimeraType;
+
+        // Default, fixed flags
+        bool calculate_reactions = false;
+        bool calculate_norm_dx_flag = true;
+
+        ModelPart& r_model_part = BaseType::GetModelPart();
+
+        bool use_slip = BaseType::UseSlipConditions();
+        // Modification of the DofSet is managed by the fractional step strategy, not the auxiliary velocity and pressure strategies.
+        bool reform_dof_set = BaseType::GetReformDofSet();
+        std::size_t echo_level = BaseType::GetEchoLevel();
+        std::size_t strategy_echo_level = (echo_level > 0) ? (echo_level-1) : 0;
+
+        if ( rStrategyLabel == BaseType::Velocity )
+        {
+            ModelPart &r_fs_velocity_model_part = r_model_part.CreateSubModelPart("fs_velocity_model_part");
+            FastTransferBetweenModelPartsProcess(r_fs_velocity_model_part, r_model_part).Execute();
+            // Velocity Builder and Solver
+            ResidualBasedBlockBuilderAndSolverPointerType p_build_and_solver =  Kratos::make_shared<ResidualBasedBlockBuilderAndSolverWithConstraintsForChimeraType>(pLinearSolver);
+
+            SchemePointerType p_scheme;
+            //initializing fractional velocity solution step
+            if (use_slip)
+            {
+                std::size_t domain_size = BaseType::GetDomainSize();
+                SchemePointerType p_temp = SchemePointerType(new ResidualBasedIncrementalUpdateStaticSchemeSlip< TSparseSpace, TDenseSpace > (domain_size,domain_size));
+                p_scheme.swap(p_temp);
+            }
+            else
+            {
+                SchemePointerType p_temp = SchemePointerType(new ResidualBasedIncrementalUpdateStaticScheme< TSparseSpace, TDenseSpace > ());
+                p_scheme.swap(p_temp);
+            }
+
+            // Strategy
+            BaseType::mStrategies[rStrategyLabel] = StrategyPointerType(new ResidualBasedLinearStrategy<TSparseSpace, TDenseSpace, TLinearSolver >
+                                                                        (r_fs_velocity_model_part, p_scheme, pLinearSolver, p_build_and_solver, calculate_reactions, reform_dof_set, calculate_norm_dx_flag));
+        }
+        else if ( rStrategyLabel == BaseType::Pressure )
+        {
+            ModelPart &r_fs_pressure_model_part = r_model_part.CreateSubModelPart("fs_pressure_model_part");
```

Comment:
maybe add the name of the parent-mp to make sure its unique?
=> I did this in the MeshMoving Strategies bcs we had problems

Reply:
done !